src/main/java/org/folio/mr/service/impl/StaffSlipContextService.java

```diff
@@ -24,68 +24,117 @@ public class StaffSlipContextService {
   public SendItemInTransitResponseStaffSlipContext createStaffSlipContext(MediatedRequest request) {
     log.debug("createStaffSlipContext:: parameters request: {}", request);
 
-    Item item = inventoryService.fetchItem(request.getItemId());
+    var staffSlipContextItem = new SendItemInTransitResponseStaffSlipContextItem();
 
-    var holding = inventoryService.fetchHolding(item.getHoldingsRecordId());
-    var instance = inventoryService.fetchInstance(holding.getInstanceId());
-    var materialType = inventoryService.fetchMaterialType(item.getMaterialTypeId());
-    var loanType = inventoryService.fetchLoanType(item.getPermanentLoanTypeId());
-    var toServicePoint = inventoryService.fetchServicePoint(item.getInTransitDestinationServicePointId());
-
-    var staffSlipContextItem = new SendItemInTransitResponseStaffSlipContextItem()
-      .title(instance.getTitle())
-      .primaryContributor(getPrimaryContributorName(instance))
-      .allContributors(getAllContributorNames(instance))
-      .barcode(item.getBarcode())
-      .status(item.getStatus().getName().getValue())
-      .enumeration(item.getEnumeration())
-      .volume(item.getVolume())
-      .chronology(item.getChronology())
-      .copy(Objects.requireNonNullElse(item.getCopyNumber(), ""))
-      .displaySummary(item.getDisplaySummary())
-      .yearCaption(String.join("; ", item.getYearCaption()))
-      .materialType(materialType.getName())
-      .loanType(loanType.getName())
-      .numberOfPieces(item.getNumberOfPieces())
-      .descriptionOfPieces(item.getDescriptionOfPieces())
-      .toServicePoint(toServicePoint.getName());
-
-    if (item.getLastCheckIn() != null) {
-      var fromServicePoint = inventoryService.fetchServicePoint(
-        item.getLastCheckIn().getServicePointId());
+    Item item = inventoryService.fetchItem(request.getItemId());
+    if (item != null) {
       staffSlipContextItem
-        .fromServicePoint(fromServicePoint.getName())
-        .lastCheckedInDateTime(item.getLastCheckIn().getDateTime());
-    }
+        .barcode(item.getBarcode())
+        .status(item.getStatus().getName().getValue())
+        .enumeration(item.getEnumeration())
+        .volume(item.getVolume())
+        .chronology(item.getChronology())
+        .copy(Objects.requireNonNullElse(item.getCopyNumber(), ""))
```

Comment:
Why are we using default value only for `copyNumber`? What about other properties?